src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -113,12 +122,12 @@ const TrackPanelListItem: FunctionComponent<TrackPanelListItemProps> = (
         <button onClick={changeDrawerViewHandler}>
           <img src={ellipsis.icon.on} alt={`Go to ${track.label}`} />
         </button>
-        <button onClick={toggleTrack}>
-          <img
-            src={trackStatus === 'on' ? eye.icon.on : eye.icon.off}
-            alt={eye.description}
-          />
-        </button>
+        <ToggleImageButton
+          buttonStatus={trackStatus}
+          description={'enable/disable track'}
+          onClick={toggleTrack}
+          image={Eye}
+        />
```

Comment:
I am not sure what TrackPanelListItem does (and now that `master` has hardcoded `http://127.0.0.1:4000` as api host I can't check it in my browser), but wouldn't you say that an intermediary component would be more suitable here?

For example, let's assume that this intermediary component is called `VisibiliryButton`. It, in turn, calls `ToggleImageButton`. This makes `VisibiliryButton` responsible for the proper icon (TrackPanelListItem shouldn't know that it's an eye) and for supplying `ToggleImageButton` with custom classes (if required) that can override default styling.